lang/java/reef-common/src/main/java/org/apache/reef/runtime/common/driver/evaluator/AllocatedEvaluatorImpl.java

```diff
@@ -136,8 +145,21 @@ public void submitContextAndServiceAndTask(final Configuration contextConfigurat
   }
 
   @Override
+  @Deprecated
   public void setType(final EvaluatorType type) {
```

Comment:
This should be deprecated on the interface as well. And there, we should have an `@deprecated` documentation as well that tells people what to migrate to.